docs/converting.asciidoc

```diff
@@ -12,41 +12,41 @@ Before you start a conversion, be sure that you understand the basics below.
 
 [float]
 [[core-or-ext]]
-=== Core and extended fields
+==== Core and extended fields
 
-* *Core fields.* Fields that are most common across all use cases are called *core fields*. 
+* *Core fields.* Fields that are most common across all use cases are called *core fields*.
 +
 These generalized fields are used by analysis content
 (searches, visualizations, dashboards, alerts, machine learning jobs, reports)
 across use cases. Analysis content designed to operate on these
-fields should work properly on data from any relevant source. 
+fields should work properly on data from any relevant source.
 +
-Focus on populating these fields first. 
+Focus on populating these fields first.
 
-* *Extended fields.* Any field that is not a core field is called an *extended field*. 
+* *Extended fields.* Any field that is not a core field is called an *extended field*.
 Extended fields may apply to more narrow use cases, or may be more open
 to interpretation depending on the use case. Extended fields are more likely to
 change over time.
 
-Each {ecs} <<ecs-fields,field>> in a table is identified as core or extended.
+Each {ecs} <<ecs-field-reference,field>> in a table is identified as core or extended.
 
 [float]
-[[ecs-comv]]
-=== An approach to mapping an existing implementation
+[[ecs-conv]]
+==== An approach to mapping an existing implementation
 
 Here's the recommended approach for converting an existing implementation to {ecs}.
 
 . Start with Core fields.
 +
 Populate core fields first. Look at your set of event fields, and find
-the appropriate ECS field name for each one. 
+the appropriate ECS field name for each one.
```

Comment:
This section needs to be clarified/simplified here and wherever else it is used.  Suggestion:

1. Review each field in your original event and map it to the relevant ECS field.

- Start by mapping your field to the relevant ECS Core field.
- If a relevant ECS Core field does not exist, map your field to the relevant ECS Extended field.
- If no relevant ECS Extended field exists, consider keeping your field with its original details, or possibly renaming it using ECS naming guidelines and attempt to map one or more of your original event fields to it.

2. Review each ECS Core field, and attempt to populate it with data from the relevant field or fields from your original event.

Reply:
Really love this. I'm pushing this to the PR right away. I've used this and added a few small but important things.

I really love how much clearer the steps are becoming :-)